spec/controllers/api/comments_controller_spec.rb

```diff
@@ -34,25 +34,32 @@
     end
 
     describe '#index' do
-      context 'when comments are created' do
+      context 'when comments are created and have status active' do
         let(:expected_hash) do
-          [{ "content" => "content comment 1",
+          [{ "id" => comment1.id,
+             "content" => "content comment 1",
              "full_name" => 'John Doe',
-             "created_at" => Time.now.to_i,
+             "created_at" => Time.now.to_i * 1000,
              "user_initials_background_color" => mobile_user.initials_background_color,
              "user_initials" => 'JD',
-             "user_id" => mobile_user.id },
-           { "content" => "content comment 2",
+             "user_id" => mobile_user.id,
+             "status" => "accepted" },
+           { "id" => comment2.id,
+             "content" => "content comment 2",
              "full_name" => 'John Doe',
-             "created_at" => Time.now.to_i,
+             "created_at" => Time.now.to_i * 1000,
              "user_initials_background_color" => mobile_user.initials_background_color,
              "user_initials" => 'JD',
-             "user_id" => mobile_user.id }]
+             "user_id" => mobile_user.id,
+             "status" => "accepted" }]
         end
 
+        let(:comment1) { build(:comment, user: mobile_user, content: 'content comment 1', debate: debate, status: :accepted) }
+        let(:comment2) { build(:comment, user: mobile_user, content: 'content comment 2', debate: debate, status: :accepted) }
```

Comment:
You can use `let!` here instead of saving records in before hook
remove `content` from comment creation